### PR TITLE
Feature/deployment

### LINKED ISSUE #4: Fix `npm run build` before deploying

### PR DESCRIPTION
**Pull request by CloudNiner**
Adds support for deploying the project to S3.

First two commits are duplicates of #6, and can be ignored here.

Closes #4 by fixing up the webpack dist build

### CODE REVIEW

File: s3_website.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+s3_id: <%= ENV['AWS_ACCESS_KEY'] %>
+s3_secret: <%= ENV['AWS_SECRET_ACCESS_KEY'] %>
+s3_bucket: adb-mongolia-nua
```
Comment (sharph) [inline s3_website.yml:3]: If you make this an environment variable, CI can push to separate buckets depending on if `master` or `develop`.

**Comment by sharph** [comment]
LGTM! Since travis is set up we might consider having it run the deployment.

:+1:

**Comment by rmartz** [comment]
LGTM

File: s3_website.yml (hunk 1 above)
Comment (CloudNiner) [inline s3_website.yml:3]: Good idea. I'll switch it to use an env variable and we can set that up once we have other buckets to deploy to.

**Pull request merged**